lib/conjur_client.rb

```diff
@@ -28,7 +28,7 @@ def policy
     end
   
     def ssl_cert
-      ENV['CONJUR_SSL_CERTIFICATE']
+      ENV['CONJUR_SSL_CERTIFICATE'] unless ENV['CONJUR_SSL_CERTIFICATE'].to_s.empty?
```

Comment:
Because you have active_support, you can say `unless ENV['CONJUR_SSL_CERTIFICATE'].blank?`

Reply:
Ah, I thought we removed it and I forgot to check. Thanks for the efficient Ruby! :)